docs/openapi/sites-api.yaml

```diff
@@ -125,3 +125,47 @@ sites-for-organization:
         $ref: './responses.yaml#/500'
     security:
       - api_key: [ ]
+bulk-sites-audit-config:
+    patch:
+      tags:
+          - site
+      summary: Enable audits for multiple sites
+      description: |
+        This endpoint is useful for enabling audit types for multiple sites.
```

Comment:
that's fine - all i'm saying is the API doc should transparently describe the effects.

Reply:
I agree to rename to make this more clear.

Specifically for enabling the org, I will add here the discussion I had with @alinarublea :
Users might not always want to enable/disable at the org level when they do this for the site.
E.g. case we had with a customer, where we had to enable the whole org and disable a part of the sites within that org. So we need a way to disable at site level without disabling at the same time at org level.

For enable = true, it's pointless to enable just the site, because anyways the audit doesn't run unless the org is enabled as well. In this case, we'd want to unify those 2 changes in 1 API call.